images/runtime/php/5.6/base.stretch.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php-5.6
+FROM oryxdevmcr.azurecr.io/private/oryx/php-5.6
```

Comment:
I mentioned something similar in a response to Stella's comment below, but even though I added a reference to `oryxdevmcr` here (and a lot of other places), we were never (and still aren't) pushing these additional images to our ACR. These are "intermediate" images that we build locally and tag in a specific way so that we can re-use them as base layers for multiple images we end up producing. The Secure Supply Chain Analysis treats anything without a registry as "being from Docker Hub", so we have to prefix all of the images with our ACR instance to avoid getting flagged for images we're not even pulling.